base_classes/NXdata.nxdl.xml

```diff
@@ -446,21 +446,47 @@
 	</field>
 
 	<!-- Data vs. plot coordinates -->
-	<field name="scaling_factor" type="NX_FLOAT">
+	<field name="FIELDNAME_scaling_factor" type="NX_FLOAT">
 		<doc>
-			The elements in data are usually float values really. For
-			efficiency reasons these are usually stored as integers
-			after scaling with a scale factor. This value is the scale
-			factor. It is required to get the actual physical value,
-			when necessary.
+			An optional scaling factor to apply to the values in FIELDNAME (usually the signal).
+
+			The elements stored in NXdata datasets are often stored as integers for efficiency
+			reasons and need further correction or conversion, generating floats.  For example,
+			raw values could be stored from a device that need to be converted to values that
+			represent the physical values. The two fields FIELDNAME_scaling_factor and
+			FIELDNAME_offset allow linear corrections using the following convention:
+
+			.. code-block::
+
+				corrected values = (FIELDNAME + offset) * scaling_factor
+
+			This formula will derive the values to use in downstream applications, when necessary.
 		</doc>
 	</field>
-	<field name="offset" type="NX_FLOAT">
+	<field name="FIELDNAME_offset" type="NX_FLOAT">
 		<doc>
-			An optional offset to apply to the values in data.
+			An optional offset to apply to the values in FIELDNAME (usually the signal).
+
+			See :ref:`FIELDNAME_scaling_factor &lt;/NXdata/FIELDNAME_scaling_factor-field&gt;` for more information.
 		</doc>
 	</field>
 
+	<field name="scaling_factor" type="NX_FLOAT" deprecated="Use FIELDNAME_scaling_factor instead">
+		<doc>
+			scaling_factor is ambiguous in the case of multiple signals, so use
```

Comment:
I think this sentence is too busy: it's trying to covey different pieces of information, which makes it somewhat hard to parse.  I tend to favour shorter sentences with more direct meaning; e.g.,

    The scaling_factor and FIELDNAME_scaling_factor fields have similar semantics.
    However, scaling_factor is ambiguous in the case of multiple signals.  Therefore
    scaling_factor is deprecated.  Use FIELDNAME_scaling_factor instead, even when
    only a single signal is present.

A similar phrasing could be used for `offset` / `FIELDNAME_offset`.

Reply:
Hi @paulmillar, I wasn't intending to explicitly deprecate sclaing_factor and offset in favor of FIELDNAME_scaling_factor and FIELDNAME_offset.  Is that something we should consider?  Comments appreciated.

Reply:
Hi @phyy-nx ,
I don't really have a strong opinion on deprecating `scaling_factor` and `offset` -- I'm mildly in favour of it, as I think it could lead to simpler analysis/viewing software; at least, in the long run.

However, I'm a bit confused by your statement:
> I wasn't intending to explicitly deprecate sclaing_factor and offset in favor of FIELDNAME_scaling_factor and FIELDNAME_offset.

I understood the proposed change does exactly this, by updating the field definition to state:
> <field [..] deprecated="Use FIELDNAME_scaling_factor instead">

Reply:
You're right, I forgot!  Applied your suggestions.